Pipfile

```diff
@@ -103,3 +103,6 @@ nodeenv = "*"
 
 [requires]
 python_version = "2.7"
+
+[pipenv]
+allow_prereleases = true
```

Comment:
Aha, so this is the magic to fix our lock file woes! :)

Reply:
I had to add this to allow it to install `le-pycaption` which I only made an alpha release. If there's concern, I could make another release.